stock_picking_invoice_link/views/account_invoice_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--
+        Stock Picking Invoice Link module for Odoo
+        Copyright (C) 2013-2014 Akretion (http://www.akretion.com/)
+        @author: Alexis de Lattre <alexis.delattre@akretion.com>
+        The licence is in the file __openerp__.py
```

Comment:
Even better if this adjusts to current template: https://github.com/OCA/maintainer-tools/blob/master/template/module/views/model_name_view.xml#L2